.github/workflows/wolfi-presubmit.yaml

```diff
@@ -157,15 +163,12 @@ jobs:
           repository: "wolfi-dev/advisories"
           path: "data/wolfi-advisories"
 
-      # this need to point to main to always get the latest action
-      - uses: wolfi-dev/actions/install-wolfictl@main # main
-
       - name: Test installable and Scan for CVEs
         run: |
           if [[ "${{ matrix.package }}" == "fping" ]]; then
-            docker run --rm -v $(pwd):/work --workdir /work cgr.dev/chainguard/wolfi-base /bin/sh -c "apk add --allow-untrusted packages/x86_64/${{ matrix.package }}-*.apk; apk add libcap-utils; getcap /usr/sbin/fping"
+            docker run --rm -v $(pwd):/work --workdir /work cgr.dev/chainguard/wolfi-base /bin/sh -c "sed 's|=.*||' -i /etc/apk/world; apk add --allow-untrusted -X ./packages/ packages/x86_64/${{ matrix.package }}-*.apk; apk add libcap-utils; getcap /usr/sbin/fping"
           else
-            docker run --rm -v $(pwd):/work --workdir /work cgr.dev/chainguard/wolfi-base apk add --allow-untrusted packages/x86_64/${{ matrix.package }}-*.apk
+            docker run --rm -v $(pwd):/work --workdir /work cgr.dev/chainguard/wolfi-base /bin/sh -c "sed 's|=.*||' -i /etc/apk/world; apk add --allow-untrusted -X ./packages/ packages/x86_64/${{ matrix.package }}-*.apk"
```

Comment:
You didn't insert this bug though, it was present before.

`packages/x86_64/${{matrix.package}}-*.apk`  won't necessarily match all packages built by package. As specific and relevant example, `glibc` builds package `ld-linux`.

I don't have an obvious solution other than just using `packages/x86_64/*.apk`. Poking through history of this workflow, it looks like we've probably never tested the installation of 'ld-linux'.